.ci/ci-setup.sh

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2017 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+set -e
+
+echo "Install chronic"
+sudo apt-get install -y moreutils
+
+echo "Install curl"
+chronic sudo apt-get install -y curl
+
+echo "Add clear containers sources to apt list"
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/xUbuntu_16.10/ /' >> /etc/apt/sources.list.d/cc-oci-runtime.list"
+
+echo "Update apt repositories"
+chronic sudo apt-get update
+
+echo "Install linux-container kernel"
+chronic sudo apt-get install -y --force-yes linux-container
+
+echo "Install qemu-lite binary"
+chronic sudo apt-get install -y --force-yes qemu-lite
+
+echo "Download clear containers image"
+latest_version=$(curl -sL https://download.clearlinux.org/latest)
+chronic curl -LO "https://download.clearlinux.org/current/clear-${latest_version}-containers.img.xz"
+
+echo "Validate clear containers image checksum"
+chronic curl -LO "https://download.clearlinux.org/current/clear-${latest_version}-containers.img.xz-SHA512SUMS"
+chronic sha512sum -c clear-${latest_version}-containers.img.xz-SHA512SUMS
+
+echo "Extract clear containers image"
+chronic unxz clear-${latest_version}-containers.img.xz
+
+cc_img_path="/usr/share/clear-containers"
+cc_img_link_name="clear-containers.img"
+chronic sudo mkdir -p ${cc_img_path}
+echo "Install clear containers image"
+chronic sudo install --owner root --group root --mode 0644 clear-${latest_version}-containers.img ${cc_img_path}
+
+echo -e "Create symbolic link ${cc_img_path}/${cc_img_link_name}"
+chronic sudo ln -fs ${cc_img_path}/clear-${latest_version}-containers.img ${cc_img_path}/${cc_img_link_name}
+
+echo "Download golang tools"
+go get github.com/fzipp/gocyclo
+go get github.com/gordonklaus/ineffassign
+go get github.com/golang/lint/golint
+go get github.com/client9/misspell/cmd/misspell
+
+echo "Setup virtcontainers environment"
+chronic sudo -E bash utils/virtcontainers-setup.sh
+
+echo "Check and build virtcontainers"
+go_packages=$(go list ./... | grep -v vendor)
```

Comment:
You might want to do what we did with cc-oci-runtime and introduce a `ci-common.sh` for things like this and source it from the other scripts as this is the 2nd time this variable has been defined (with the same value).